### PR TITLE
[WIP] Manual.md: document more variables

### PR DESCRIPTION
**Pull request by maxice8**
See commits, they are well described.

This is a work in progress.

@Cogitri

### CODE REVIEW

**Comment by maxice8** [comment]
> Maybe it'd make sense to first describe what host&target is since we sort of do it in a way that no one else does. I think meson describes it pretty well: https://mesonbuild.com/Cross-compilation.html

No clue where that should go and if we should have an entire section or small notes.

**Comment by maxice8** [comment]
added XBPS_CROSS_BASE

File: Manual.md [hunk 1]
```diff
@@ -356,6 +356,8 @@ in this directory such as `${XBPS_BUILDDIR}/${wrksrc}`.
 
 - `XBPS_CROSS_TRIPLET` The target machine GNU type triplet, only available when cross-compiling.
 
+- `XBPS_CROSS_BASE` Full path to where dependencies for cross compilation are installed.
```
Comment (Cogitri) [inline Manual.md]: Somehow this doesn't really explain where it actually points to, maybe an example (`/usr/${XBPS_CROSS_TRIPLET}`) would make that clearer

**Comment by maxice8** [comment]
On Sat, Oct 13, 2018 at 05:05:22AM -0700, Rasmus Thomsen wrote:
> Cogitri commented on this pull request.
> 
> 
> 
> > @@ -356,6 +356,8 @@ in this directory such as `${XBPS_BUILDDIR}/${wrksrc}`.
>  
>  - `XBPS_CROSS_TRIPLET` The target machine GNU type triplet, only available when cross-compiling.
>  
> +- `XBPS_CROSS_BASE` Full path to where dependencies for cross compilation are installed.
> 
> Somehow this doesn't really explain where it actually points to, maybe an example (`/usr/${XBPS_CROSS_TRIPLET}`) would make that clearer
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/3699#pullrequestreview-164461008

Sadly there is no way to really explain it without telling people all about cross compilation

**Pull request closed without merging**